cohorts/cohort.py

```diff
@@ -478,32 +477,23 @@ def _load_single_patient_variants(self, patient, filter_fn):
 
         if len(variant_collections) == 1:
             # There is nothing to merge
-            vcf_path, variants = variant_collections[0]
+            variants = variant_collections[0]
             merged_variants = variants
         else:
-            merged_variants = self._merge_variant_collections(dict(variant_collections), self.merge_type)
+            merged_variants = self._merge_variant_collections(variant_collections, self.merge_type)
 
         self.save_to_cache(merged_variants, self.cache_names["variant"], patient.id, cached_file_name)
         return filter_variants(variant_collection=merged_variants,
                                patient=patient,
                                filter_fn=filter_fn)
 
-    def _merge_variant_collections(self, vcf_to_variant_collections, merge_type):
+    def _merge_variant_collections(self, variant_collections, merge_type):
         assert merge_type in ["union", "intersection"], "Unknown merge type: %s" % merge_type
-        combined_variants = [set(vc.elements) for (path, vc) in vcf_to_variant_collections.items()]
+        head = variant_collections[0]
```

Comment:
`variant_collections[0]` is a `varcode.VariantCollection`. Does it say otherwise somewhere?